### PR TITLE
Build /compare page

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

Build `/compare` page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8027/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Please check page against [copy doc](https://docs.google.com/document/d/1wWos3iWMTCqDbxZBDuSVqvY6ZwYux-goCrX0L0E5Weo/edit)

## Issue / Card

Fixes [#4649](https://github.com/canonical-web-and-design/web-squad/issues/4649)

## Screenshots
MicroK8s vs k3s vs Minikube ()

### CODE REVIEW

**Comment by minkyngkm** [comment]
@alxchalkias Is navigation okay? Or do you want to move "compare" to somewhere else? 
image

**Comment by alxchalkias** [comment]
+1 thank you.

File: templates/compare/index.html [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+          <th class="u-align--center">MicroK8s</th>
+          <th class="u-align--center">K3s</th>
+          <th class="u-align--center">minikube</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr>
+          <td aria-label="Category">CNCF certified</td>
+          <td aria-label="MicroK8s" class="u-align--center">
+            <img src="https://assets.ubuntu.com/v1/f1a7515d-tick-darkaubergine.svg" alt="Yes">
```
Comment (sowasred2012) [inline templates/compare/index.html:41]: I'm pretty sure we decided to start phasing out the aubergine tick icon, in favour of the green one available in Vanilla. See the table on https://ubuntu.com/security/cis, under the heading "Which versions of Ubuntu have CIS tooling?":

```suggestion
            <i class="p-icon--success">Yes</i>
```

File: templates/compare/index.html [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+            <a href="https://www.brighttalk.com/webcast/6793/519512?utm_source=brighttalk-portal&utm_medium=web&utm_content=canonical&utm_term=search-result-7&utm_campaign=webcasts-search-results-feed">MicroK8s on IBM Z — minimal footprint meets zero downtime</a>
+          </li>
+        </ul>
+     </div>
+    </div>
+    <div class="col-4 p-divider__block">
+      <div class="p-heading-icon--small">
+        <div class="p-heading-icon__header">
+          <img class="p-heading-icon__img" src="https://assets.ubuntu.com/v1/6e184942-Webinar.svg" alt="">
+          <h3 class="p-heading--4 p-heading-icon__title">Webinars on demand</h3>
```
Comment (sowasred2012) [inline templates/compare/index.html:232]: @alxchalkias it looks a little odd with two headers on this section being identical - what do you think of doing this instead?
Screenshot from 2022-01-26 14-34-26

**Review by sowasred2012 — CHANGES_REQUESTED** [review]
Looks good! Just the tick icons I think need changing - and a question for @alxchalkias

File: templates/compare/index.html (hunk 2 above)
Comment (alxchalkias) [reply templates/compare/index.html:232]: +1, this is much better

**Comment by minkyngkm** [comment]
@sowasred2012 Thanks Scott for reviewing! I've updated the tick icons :)

**Comment by alxchalkias** [comment]
Sorry I missed this before, the heading should read "MicroK8s and Kubernetes resources" not just "Kubernetes resources". You can merge the PR after that last change.

File: templates/compare/index.html [hunk 3]
```diff
@@ -0,0 +1,279 @@
...
+        <p>With MicroK8s you get a complete, CNCF-compliant Kubernetes environment and all the services you need to run your OCI containers at scale, such as networking & load balancing, storage, service mesh, observability, GPU & FPGA acceleration, multi-cluster management and more - something unique among lightweight Kubernetes distributions.</p>
+      </div>
+      <div class="col-5 u-align--center u-vertically-center u-hide--small u-hide--medium">
+        <img src="https://assets.ubuntu.com/v1/990738e2-kubernetes-cloud.svg" alt="" width="263" height="150" loading="lazy" />
+      </div>
+    </div>
+  </section>
+<section class="p-strip--light">
+  <div class="u-fixed-width">
+    <h2>Kubernetes resources</h2>
```
Comment (sowasred2012) [inline templates/compare/index.html:202]:
```suggestion
    <h2>MicroK8s and Kubernetes resources</h2>
```

**Pull request merged**